creator/jobs/schema.py

```diff
@@ -1,9 +1,10 @@
 import django_filters
+from django_filters import FilterSet, OrderingFilter
```

Comment:
Do you need to re-order these imports here?

Reply:
nope. i like having them order alphabetically